### ISSUE TITLE
Only works once

### ISSUE THREAD

**Issue by groschatchauve** [tksugimoto/chrome-extension_open-local-file-link#3]
Hi,
Thanks for this usefull extension.
When I set up the extension frome the Google Store, it works fine : I can get local file and local folder.
But if I close Chrome and start again Chrome. It does not work anymore.
The extension is still set up and active.
I have this problem on two PC, running the version 59 of Chrome.
I have not try on another PC.
In the browser console, the famous message saying that we cannot load local file is written.

**Comment by tksugimoto** [comment]
Hi,

Sorry, I can't reproduce the error. And I have never heard of that symptom.
Chrome 60 (now)

Perhaps it seems that it interfered with other Chrome extensions.
When I executed the following code on the page, this extension became ineffective and the same symptoms occurred.
Are there extensions that are doing something similarly installed?

```js
document.body.addEventListener("click", evt => {
    console.log("body clicked", evt);
    evt.stopPropagation();
}, true);
```

----

This extension's functions.

* Register the event so that `file: //` judgment will be done when link is clicked on all open pages when this extension is installed.
* When pages are ready for installation after the installation, event registration is done so that `file: //` judgment is done when link clicking.
  * I think that this behavior is the same as after restarting Chrome.
  * After installing this extension, did you open the local file link of the opened page before restarting Chrome?

**Comment by groschatchauve** [comment]
Thank you for your response.
At the moment, I cannot update to Chrome 60 due to company policy.

So there are two clues 
- Extension Script Conflict
- WebSite Script Conflict

There is one Amazon Web Service based extension set up by my company : [WalkMe](http://help.walkme.com/wp-content/uploads/2014/07/WalkMe-Extensions.pdf).
This extension may be the problem here but unfortunately I cannot desactivate it.

I have observed that when I use my website as standalone I can open local link.
But when I use it as a frame, I cannot open it. So I am quite sure that it is not an extension problem but I am not 100% sure.
Anyway, I have checked JavaScript with the browser console and I see a lot of functions based on "click" event.
Is there a way to make your function more important than the others (as "!important" in CSS).

> After installing this extension, did you open the local file link of the opened page before restarting 
 Chrome?

There are several links, I clicked on some before restarting and I do not clicked on some others before restarting. It is the same result.

**Comment by tksugimoto** [comment]
> the extension does not work with frame at the moment. 

Sorry, I forgot frame page support except at this extension is installed.
I appreciate the investigation and report.

Now, I released version `0.5.0` that supports frame pages.
It will be delivered soon.

If you still have problems, please ask me again.

**Comment by groschatchauve** [comment]
I download it again on the Chrome Store and it works fine now.

Thank you for your work on this usefull extension!
